pkg/agent/flowexporter/flowrecords/flow_records.go

```diff
@@ -109,6 +110,19 @@ func (fr *FlowRecords) GetFlowRecordFromMap(connKey *flowexporter.ConnectionKey)
 	return &record, exists
 }
 
+// GetFlowRecordFromMap gets the flow record from record map given connection key.
+// This is used only for unit tests.
+func (fr *FlowRecords) DeleteFlowRecordFromMap(connKey *flowexporter.ConnectionKey) error {
+	fr.mutex.Lock()
+	defer fr.mutex.Unlock()
+	_, exists := fr.recordsMap[*connKey]
+	if !exists {
+		return fmt.Errorf("record with key %v doesn't exist in map", connKey)
+	}
+	delete(fr.recordsMap, *connKey)
+	return nil
+}
```

Comment:
Shall we pass `connKey` value instead of pointer? Because we will convert it to value anyways in this function.

Reply:
Done